website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -3,102 +3,57 @@
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-</Doc::Layout>
-
-Use color logically.
-
-- **Neutral** to provide general information to the user about an ongoing process.
-- **Highlight** use interchangeably with `neutral` when more prominence is needed. Use it sparingly.
-- **Success** to indicate a successful action was completed.
-- **Warning** to indicate a successful action was completed but may have triggered a related issue. Provide guidance and actions if possible.
-
-- **Critical** to indicate error or critical issues resulting from a failed action.
+![A list different types of toasts, starting from neutral all the way to critical](/assets/components/toast/toast-colors.png)
+
+Use color logically:
+
+- **Neutral**: provide general information to the user about an ongoing process.
+- **Highlight**: used interchangeably with `neutral` when more prominence is needed. Use sparingly.
+- **Success**: indicate a successful action was completed.
+- **Warning**: indicate a successful action was completed but may have triggered a related issue. Provide guidance and actions if possible.
+- **Critical**: indicate error or critical issues resulting from a failed action.
 
 ### Critical toasts
 
 !!! Do
 
 Use the Toast to provide non-intrusive feedback to users about the failure of an ongoing task or request. For example, a failure while deleting a cluster.
 
-<Hds::Toast @color="critical" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Module error</T.Title>
-    <T.Description>This module encountered an error during publishing. You may need to republish.</T.Description>
-  </Hds::Toast>
+![A toast showing a module error](/assets/components/toast/toast-do-non-intrusive-notification.png)
+
 !!!
 
 !!! Do
 
 Use the Toast to communicate error messages that are not caused by the user. For example, an unsuccessful Vault cluster creation due to a failure while validating the deployment.
 
-<Hds::Toast @color="critical" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Vault cluster update failed</T.Title>
-    <T.Description>**test-cluster_1** configuration update failed.</T.Description>
-  </Hds::Toast>
+![A toast showing a vault cluster update failure](/assets/components/toast/toast-do-system-notifications.png)
+
 !!!
 
 !!! Dont
 
 Don't use toasts for intrusive message communication about errors or critical disruptions at an application, page, or section level. Use the [Alert](/components/alert) instead.
 
-<Hds::Toast @color="critical" @onDismiss={{this.noop}} as |T|>
-    <T.Title>You have exceeded 50 applies this month</T.Title>
-    <T.Description>You may only invoke applies that destroy managed resources. Upgrade now and access additional product features, unlimited applies, and increased concurrency.
-    </T.Description>
-  </Hds::Toast>
+![A toast with an intrusive message](/assets/components/toast/toast-dont-intrusive-message.png)
```

Comment:
```suggestion
![A toast showing the user has exceeded their applies limit.](/assets/components/toast/toast-dont-intrusive-message.png)
```

Just because it is useful to know an example of an intrusive message.